codingTest_김준우/k번째 수.js

```diff
@@ -0,0 +1,19 @@
+function solution(array, commands) {
+  let i;
+  let j;
+  let k;
+  let ArrNum = commands.length;
+  let answer = 0;
+  const arr = new Array();
+  for (let a = 0; a < ArrNum; a++) {
+    i = commands[a][0];
+    j = commands[a][1];
+    k = commands[a][2];
+    let array1 = array.slice(i - 1, j);
+    array1 = array1.sort((a, b) => a - b);
```

Comment:
```suggestion
    array1.sort((a, b) => a - b);
```
굳이 이렇게 따로 할당안해도 sort를 돌리면 정렬된 배열로 변합니다.